action/action-util/src/main/java/com/powsybl/action/util/PhaseShifterShiftTap.java

```diff
@@ -8,22 +8,22 @@
 
 import com.powsybl.commons.PowsyblException;
 import com.powsybl.computation.ComputationManager;
-import com.powsybl.contingency.tasks.ModificationTask;
 import com.powsybl.iidm.network.Network;
 import com.powsybl.iidm.network.PhaseTapChanger;
 import com.powsybl.iidm.network.TwoWindingsTransformer;
+import com.powsybl.network.modification.NetworkModification;
 
 import java.util.Objects;
 
 /**
  * @author Hamou AMROUN <hamou.amroun at rte-france.com>
  */
-public class PhaseShifterTapTask implements ModificationTask {
+public class PhaseShifterShiftTap implements NetworkModification {
```

Comment:
Why has this been renamed `PhaseShifter**Shift**Tap`?

Reply:
Same standardizing try, as `PhaseShifterTap` alone does not indicate what kind of modification is done.